maps/site53/job/jobs/command.dm

```diff
@@ -57,6 +81,19 @@ ut // COMMAND
 
 	minimal_access = list()
 
+	min_skill = list(SKILL_BUREAUCRACY = SKILL_MASTER,
+					SKILL_FINANCE      = SKILL_TRAINED,
+					SKILL_HAULING      = SKILL_TRAINED,
+					SKILL_COMBAT      = SKILL_TRAINED,
+					SKILL_WEAPONS     = SKILL_BASIC,
+					SKILL_EVA          = SKILL_BASIC,
+					SKILL_PILOT        = SKILL_BASIC)
+
+	max_skill = list(   SKILL_COMBAT      = SKILL_MASTER,
+	                    SKILL_WEAPONS     = SKILL_MASTER)
+
+
+
```

Comment:
See same comment for SD.

No.

Reply:
Seems like PRs are being made to support the inevitability of a CD or SCP breach- why would the SD have any master skills other than burecracy & finance?